### PR TITLE
Added Validation for delete segments query, load start time parameter

### PR DESCRIPTION
**Pull request by ManoharVanam**
Added Validation for delete segments query, load start time parameter

### CODE REVIEW

File: core/src/main/java/org/carbondata/core/load/LoadMetadataDetails.java [hunk 1]
```diff
@@ -126,6 +139,33 @@ public String getLoadStartTime() {
...
+  /**
+   * returns load start time as long value
+   * @param loadStartTime
+   * @return
+   */
+  private Long getTimeStamp(String loadStartTime) {
+    if (loadStartTime.isEmpty()) {
+      return null;
+    }
+    SimpleDateFormat parser = new SimpleDateFormat(CarbonCommonConstants.CARBON_TIMESTAMP);
```
Comment (ravipesala) [inline core/src/main/java/org/carbondata/core/load/LoadMetadataDetails.java]: it could be static and class level attribute

**Comment by ravipesala** [comment]
LGTM except one small review comment. Please fix it I will merge it.

**Comment by gvramana** [comment]
LGTM

File: core/src/main/java/org/carbondata/core/load/LoadMetadataDetails.java (hunk 1 above)
Comment (ManoharVanam) [inline core/src/main/java/org/carbondata/core/load/LoadMetadataDetails.java]: fixed.

**Pull request merged**